index.html

```diff
@@ -21,10 +21,10 @@
         <h1><span class="span-color">x</span>PressUi..</h1>
         <p class="landing-content">Best CSS Component 
         library to make your web applications faster</p>
-        <a href="/components/getstarted.html"><button 
+        <a href="/componenets/getstarted.html"><button 
         class="landing-content btn-index">Get Started</button></a>
-        <a href="https://github.com/rov9/xPress.Ui"><button 
-        class="landing-content btn-index">Github</button></a>
+        <a href="https://myblog4you.hashnode.dev/documentation"><button 
+        class="landing-content btn-index">Documentation</button></a>
         </div>
     
 </body>
```

Comment:
You should use Prettier or any code formatter to make the code properly indented, it's not very readable right now